data/Makefile.am.inc

```diff
@@ -36,7 +36,7 @@ edit = sed \
 	-e 's|@instrumentationcachedir[@]|$(instrumentationcachedir)|g' \
 	$(NULL)
 
-tmpfilesddir = $(libdir)/tmpfiles.d/
+tmpfilesddir = $(prefix)/lib/tmpfiles.d/
```

Comment:
I don't see how this will break provided systemd's prefix is the same prefix which is provided by the packaging - I was just looking to implement the "common" solution that showed up under https://github.com/search?l=Makefile&q=org%3Aendlessm+tmpfiles&type=Code - for example, systemd itself defines the dir as $(prefix)/lib/tmpfiles.d/ as well.